cinema/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=100)
+    description = serializers.CharField(required=True, max_length=1000)
+    duration = serializers.IntegerField(required=True)
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("title", instance.title)
+        instance.description = validated_data.get(
+            "description",
+            instance.description
+        )
+        instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
+
+    class Meta:
+        model = Movie
+        fields = "__all__"
```

Comment:
Using "__all__" is convenient overall, but it would be safer to list all the fields to avoid exposing unintended fields if the model changes